content/docs/1.5.0/deploy/upgrade/_index.md

```diff
@@ -9,6 +9,26 @@ Here we cover how to upgrade to the latest Longhorn from all previous releases.
 
 There are no deprecated or incompatible changes introduced in v{{< current-version >}}.
 
+# Upgrade Path Enforcement
+
+Since Longhorn v1.5.0, we enforce an upgrade path. Enforce an upgrade path to prevent users from upgrading from any unsupported version. After rejecting the user's upgrade attempt, the user's Longhorn setup should remain intact without any impacts.
+
+Compare current Longhorn version with upgrading/downgrading Longhorn version, only allow authorized version upgrade (e.g., 1.3.x to 1.5.x is not allowed) as following table.
+
+  |  Current version |  Upgrading version |  Allow |
+  |    :-:      |    :-:      |   :-:  |
+  |  x.y.*      |  x.(y+1).*  |   ✓    |
+  |  x.y.0      |  x.y.*      |   ✓    |
```

Comment:
Actually we also support x.y.* to x.y.(*+n) not just from `x.y.0`.

Reply:
Modified. Thanks.